assets/ui/EnvironmentInfoScreen.ui

```diff
@@ -0,0 +1,101 @@
+{
+  "type": "EnvironmentInfoScreen",
```

Comment:
This shouldn't be called `EnvironmentInfoScreen` - go for something like `RandomNumberScreen` instead (also applies to this file's name and the Java class name). I'd also remove the "environment info" code and visuals lifted straight from the tutorial - leave the stuff you've added yourself, though 🙂